### PR TITLE
Finalize event wizard with step 3

### PR DESCRIPTION
**Pull request by Sartonon**
## Description

- Add third step to the wizard

- Add event preview to the third step

### Closes
**[PT-527](https://helsinkisolutionoffice.atlassian.net/browse/PT-527):**

### CODE REVIEW

File: src/domain/event/eventPreviewCard/EventPreviewCard.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+const EventCard: React.FC<Props> = ({ event, link }) => {
+  const { t } = useTranslation();
+  const locale = useLocale();
+
+  const id = event.id;
+  const name = getLocalisedString(event.name, locale);
+  const description = getLocalisedString(event.description || {}, locale);
+  const image = event.images[0]?.url;
+  const time = getEventStartTimeStr(event, locale, t);
```
Comment (jorilindell) [inline src/domain/event/eventPreviewCard/EventPreviewCard.tsx:29]: You could use getEventFields helper function to get these values

**Review by jorilindell — APPROVED** [review]
Looks good. Added couple of minor comments

File: src/domain/event/EventSummaryPage.tsx [hunk 2]
```diff
@@ -137,70 +148,75 @@ const OccurrencesPage: React.FC = () => {
...
-                  <div className={styles.buttonWrapper}>
-                    <Button onClick={goToEventDetailsPage} variant="secondary">
-                      {t('occurrences.buttonEventDetails')}
-                    </Button>
-                  </div>
-                </div>
-                <div className={styles.titleRow}>
-                  <h2>{t('occurrences.titleEventPublishment')}</h2>
+              ) : (
+                <BackButton onClick={() => history.push(ROUTES.HOME)}>
```
[the remark this answers is not in the document]
Comment (Sartonon) [reply src/domain/event/EventSummaryPage.tsx]: fixed :)

File: src/domain/event/EventSummaryPage.tsx [hunk 3]
```diff
@@ -271,20 +326,16 @@ const OccurrencesPage: React.FC = () => {
...
-      >
-        {t('occurrences.buttonCreateOccurrence')}
-      </Link>
-    </div>
+    <Link className={styles.link} to={link}>
+      <IconPen />
+      {text}
+    </Link>
   );
 };
```
[the remark this answers is not in the document]
Comment (Sartonon) [reply src/domain/event/EventSummaryPage.tsx]: fixed :)

File: src/domain/event/eventPreviewCard/EventPreviewCard.tsx (hunk 1 above)
Comment (Sartonon) [reply src/domain/event/eventPreviewCard/EventPreviewCard.tsx]: goot catch, fixed :)

File: src/domain/event/eventForm/EventForm.tsx [hunk 4]
```diff
@@ -424,7 +424,13 @@ const EventForm = <T extends FormFields>({
...
                   {t('eventForm.buttonCancel')}
                 </Button>
-                <Button type="submit">{t('eventForm.buttonSave')}</Button>
+                {edit ? (
+                  <Button type="submit">{t('eventForm.buttonSave')}</Button>
+                ) : (
+                  <Button type="submit">
+                    {t('eventForm.buttonSaveAndGoToOccurrences')}
+                  </Button>
+                )}
```
[the remark this answers is not in the document]
Comment (Sartonon) [reply src/domain/event/eventForm/EventForm.tsx]: good catch, fixed :)

**Pull request merged**